edisgo/grid/network.py

```diff
@@ -1282,6 +1284,12 @@ def __init__(self, edisgo_object, **kwargs):
         # check if curtailment exceeds feed-in
         self._check_curtailment(edisgo_object.network)
 
+        # write curtailment to results to be able to put it out as files for
+        # result checking
+        # make sure you don't overwrite existing curtailment data
+        edisgo_object.network.results.assigned_curtailment =\
```

Comment:
I would like to save the curtailment as we discussed, in the form of a dictionary (keys are columns of given curtailment dataframe and values are the curtailed generators). The getter should return the dictionary values in the form of a dataframe containing the curtailment time series instead of the list of generators.

Reply:
The changes will be pushed with the PR of branch test/curtailment_results